bin/run-tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
+
+# Synopsis:
+# Test the representer by running it against a predefined set of solutions 
+# with an expected output.
+
+# Output:
+# Outputs the diff of the expected representation and mapping against the
+# actual representation and mapping generated by the representer.
+
+# Example:
+# ./bin/run-tests.sh
+
+exit_code=0
+
+# Iterate over all test directories
+for test_dir in tests/*; do
+    test_dir_name=$(basename "${test_dir}")
+    test_dir_path=$(realpath "${test_dir}")
+
+    bin/run.sh "${test_dir_name}" "${test_dir_path}/" "${test_dir_path}/"
```

Comment:
I'm not sure that this is a problem, but according to the [representer docs](https://exercism.org/docs/building/tooling/representers/interface) the representer can expect to be called with `slug`, `path/to/src`, `path/to/output`. This script breaks that interface, as it passes the name of the test directory as if it's a slug.

Reply:
True, but it usually doesn't matter in practice (and it doesn't here I think).

Reply:
Fair enough!